src/components/Searchbar.css

```diff
@@ -4,4 +4,56 @@ input{
     margin-top: 2vh;
     font-size: 1.5rem;
     border: 1px solid black;
+}
+
+h1{
+	color: #fff;
+	font-weight: 400px;
```

Comment:
font-weight is not in pixel. just:

    font-weight: 400;

or, since 400 is a regular size (JTS 700 is bold):

    font-weight: regular;

or, to be synchronised with context:

    font-weight: inherit;